son/serialize.py

```diff
@@ -0,0 +1,29 @@
+"""son.serialize
+
+The serialization interface for son, defining how objects
+are turned into strings, and vice-versa. The dumper and
+loader are expected to be callables with arbitrary keyword
+arguments, which are simply passed through.
+
+"""
+
+import warnings
+
+try:
+    import ujson as json
+except ModuleNotFoundError:
+    import json
+
+
+def dump(obj, dumper=json.dumps, **kwargs):
```

Comment:
I wonder if we want to keep the name `dumper`, it sound .. dumb.

Reply:
I think it's fine, it mirrors the language of `json`. Don't see a reason to break the API over this.